### ISSUE TITLE
Message formatting doesn't work for some markup.

### ISSUE THREAD

**Issue by RIKIKU** [jgigler/Powershell.Slack#5]
Convertto-JSON seems to be taking the special characters in the output of New-SlackRichNotification and converting them into Unicode, eg. "<" becomes "\u003c" or "\n" (which is the new line indicator) becomes "\\n". 

When slack receives these Unicode characters, it converts them back to their symbols but not before they have lost all meaning to slack; they become just Text. This means that if you wanted to communicate with a specific team member by using <@UserName>, then it will become \u003c@UserName\u003e in the JSON payload which is treated like text and not like a message to a specific user. In the chat window however the special characters will have been converted back to their original symbols.

**Comment by RIKIKU** [comment]
For examples of how to format rich text messages in markup for slack see this link: https://api.slack.com/docs/formatting

**Comment by RIKIKU** [comment]
I have fixed this in my latest Master. waiting on another pull request before I can make a pull for this issue.
